modules/checking-cluster-resources-after-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-cluster-resources-after-initiating-limited-live-migration_{context}"]
+= Checking cluster resources after initiating the limited live migration
+
+The following procedure shows you how to check metrics for EgressIPs resources, EgressFirewalls resources, and multicast enabled namespaces. These resources are migrated to OVN-Kubernetes, however, they might behave differently after migration. You should check these resources after migration to ensure the intended behavior.
```

Comment:
the following steps only check egressfirewalls,  but not egressips ans multicast resource?

Reply:
@zhaozhanqi Yeah sorry. [In the question above](https://github.com/openshift/openshift-docs/pull/78178#discussion_r1688604557) I asked if we should include steps to check for egressIP/multicast resources after migration. Is that something we should do?